### ISSUE TITLE
Intermittent connectivity issues illustrated by ping failure

### ISSUE THREAD

**Issue by vsajip** [tailscale/tailscale#986]
**Describe the bug**

Connectivity seems to be intermittently lost between machines, at least at the level of pings.

1. Computer A tries to ping computer C, and the pings don't reach it.
2. Computer A connects to computer B using SSH. From the SSH terminal session on computer B, pinging computer C works.  
3. Computer A disconnects from computer B by logging out the SSH session. Immediately thereafter, A pings C without issues.

This may or may not be relevant - B and C are in the same physical location, whereas A is at a separate location.

After a while, the connectivity A - C disappears again, and can be re-invigorated by going via B again.

**To Reproduce**

Not sure you can reproduce it, as it's intermittent, but you can try to replicate the above scenario. Then, just use ping.

**Expected behavior**

Pings always work as long as the computers are up and running.

**Version information:**

Computer A - linux (Mint Ulyana, VM) - Tailscale 1.2.8
Computer B - linux (Mint Tessa, VM) - Tailscale 1.2.8
Computer C - Raspberry Pi running Raspberry Pi OS, and Tailscale 1.2.8.

**Additional context**

Console output covering above sequence of events:

```
vinay@client:~$ ping dest-machine
PING dest-machine (100.113.139.122) 56(84) bytes of data.
^C
--- dest-machine ping statistics ---
9 packets transmitted, 0 received, 100% packet loss, time 8181ms

vinay@client:~$ ssh other-machine

18 packages can be updated.
0 updates are security updates.

Last login: Wed Dec  2 22:31:42 2020 from 100.64.102.97
~  $ ping dest-machine
PING dest-machine (100.113.139.122) 56(84) bytes of data.
64 bytes from dest-machine (100.113.139.122): icmp_seq=1 ttl=64 time=1454 ms
64 bytes from dest-machine (100.113.139.122): icmp_seq=2 ttl=64 time=431 ms
64 bytes from dest-machine (100.113.139.122): icmp_seq=3 ttl=64 time=16.8 ms
64 bytes from dest-machine (100.113.139.122): icmp_seq=4 ttl=64 time=12.1 ms
64 bytes from dest-machine (100.113.139.122): icmp_seq=5 ttl=64 time=6.30 ms
^C
--- dest-machine ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4029ms
rtt min/avg/max/mdev = 6.307/384.286/1454.963/559.444 ms, pipe 2
~  $ logout
Connection to 100.80.140.25 closed.
vinay@client:~$ ping dest-machine
PING dest-machine (100.113.139.122) 56(84) bytes of data.
64 bytes from dest-machine (100.113.139.122): icmp_seq=1 ttl=64 time=60.9 ms
64 bytes from dest-machine (100.113.139.122): icmp_seq=2 ttl=64 time=55.6 ms
64 bytes from dest-machine (100.113.139.122): icmp_seq=3 ttl=64 time=57.0 ms
64 bytes from dest-machine (100.113.139.122): icmp_seq=4 ttl=64 time=66.0 ms
^C
--- dest-machine ping statistics ---
4 packets transmitted, 4 received, 0% packet loss, time 3006ms
rtt min/avg/max/mdev = 55.637/59.900/66.016/4.021 ms
```

**Comment by britannic** [comment]
I having  issues with one way tailscale pings. Server A pings B without issue, but B cannot ping back. There are no firewall rules that prevent ping over tailscale. I have only had this issue since upgrading tailscale to:
1.8.5
  tailscale commit: 118cf0cd75f92bbe3d994a5b1587c9ea3553221b
  other commit: c6fd9c9d5657e82c2d7888804ebce0e8bd6a4556
  go version: go1.16.4-ts1b677aa

**Comment by DentonGentry** [comment]
The issue when initially filed was an one way connectivity problem fixed in release 1.4.6 (I think).

The update in May was due to an issue in DERP, which was fixed in June.
